### ISSUE TITLE
Missing cut-out in FDM-0020-00_up-camera-mount

### ISSUE THREAD

**Issue by JudgeBeeb** [sphawes/index#68]
The version of FDM-0020-00_up-camera-mount in the 'freecad' is missing the cut-out for the USB header pins. I have corrected this in Fusion 360 (STEP attached), but FreeCAD continues to baffle me.

[FDM-0020-00_up-camera-mount_corrected.zip](https://github.com/sphawes/index/files/5684579/FDM-0020-00_up-camera-mount_corrected.zip)

**Comment by sphawes** [comment]
fixed in freecad branch with [this commit](https://github.com/sphawes/index/commit/11b63017c7bbad9a41de616c02dd4e4565cb5ee0)! i believe this issue will auto-close once freecad is merged into master.
